util/status.go

```diff
@@ -12,11 +12,15 @@ func StatusUnchanged(existing, new []v1.LoadBalancerIngress) bool {
 	if len(existing) != len(new) {
 		return false
 	}
+
+	sortLoadBalancerIngress(existing)
```

Comment:
I think this should also sort `new`, as there would be no guarantee that it comes pre-sorted